### ISSUE TITLE
`v3-apply-manifest` does not correctly set env vars for multi-app manifests

### ISSUE THREAD

**Issue by andy-paine** [cloudfoundry/cli#1596]
- [x] I tried updating to the latest version of the CF CLI to see if it fixed my problem.
- [x] I attempted to run the command with `CF_TRACE=1` to help debug the issue.
- [x] I am reporting a bug that others will be able to reproduce.

**Pushing a multi-app manifest with `env` section defined in each results in duplication of 1st app's env vars using `v3-apply-manifest`**
I was attempting to apply the manifest to set the environment variables (amongst other things) for a multi-app deployment using `v3-apply-manifest`.

**What happened**
There were 2 applications in my manifest and `cf env` showed that the second application had received the env vars that were configured for the first.

**Expected behavior**
Each application should get the environment variables defined in the `env` section of that app in the manifest.

**To Reproduce**
Using the following manifest:
```
applications:
  - name: app-1
    buildpacks:
      - staticfile_buildpack
    env:
      ENV_VAR: var-1

  - name: app-2
    buildpacks:
      - staticfile_buildpack
    env:
      ENV_VAR: var-2
```
1. Run `cf v3-create-app app-1`
2. Run `cf v3-create-app app-2`
3. Run `cf v3-apply-manifest -f manifest.yml`
```
This command is in EXPERIMENTAL stage and may change without notice

Applying manifest manifest.yml in org some-org / space tmp as my.cf.user@email.com...
OK
```
4. Run `cf env app-2`
```
User-Provided:
ENV_VAR: var-1
```

**Provide more context**
- Mac OSX, iTerm2, zsh
- `cf version 6.43.0+815ea2f3d.2019-02-20`
- `2.133.0`
- CF deployment `v7.7.0`

**Comment by bwasmith** [comment]
Thank you for submitting an issue @andy-paine , I have reached out the the V3 team to see what they think. 

@cloudfoundry/v3-acceleration-team-vat , this looks like something you might have some insight on?

**Comment by ewrenn8** [comment]
Hi @andy-paine, this bug has been fixed (25e252b & 59e3abb), and should be included in the next V6 release. I'm going to go ahead and close this issue. Thanks for reaching out!

Eli

**Comment by andy-paine** [comment]
Yep all seems to work using the CLI built from master 👍 Thanks for your help!
